cit3.0-web/src/components/Page/AddOpportunity/AddOpportunity.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from "prop-types";
+import { useHistory } from "react-router-dom";
+import { Container, Row, Col } from "react-bootstrap";
+import { useDispatch, useSelector } from "react-redux";
+import PortalHeader from "../../Headers/PortalHeader/PortalHeader";
+import NavigationHeader from "../../Headers/NavigationHeader/NavigationHeader";
+import MapContainer from "../../MapContainer/MapContainer";
+import AddressSearchBar from "../../AddressSearchBar/AddressSearchBar";
+import { getAddressData } from "../../../helpers/resourceCalls";
+import PropertyInfo from "../../PropertyInfo/PropertyInfo";
+import PageTitleHeader from "../../Headers/PageTitleHeader/PageTitleHeader";
+import ButtonRow from "../../ButtonRow/ButtonRow";
+import {
+  setAddress,
+  setCoords,
+  setNearbyResources,
+  setResourceIds,
+} from "../../../store/actions/opportunity";
+
+export default function AddOpportunity({ match }) {
+  const dispatch = useDispatch();
+  const address = useSelector((state) => state.opportunity.address);
+  const coords = useSelector((state) => state.opportunity.coords);
+  const nearbyResources = useSelector(
+    (state) => state.opportunity.nearbyResources
+  );
+
+  const history = useHistory();
+  const title1 = "Add an Opportunity";
+  const title2 = "Confirm Property";
+  const text1 =
+    "Please drop a min on the map or enter the address of the property you want to list as an opportunity for investors.";
```

Comment:
I think this is a typo
```suggestion
    "Please drop a pin on the map or enter the address of the property you want to list as an opportunity for investors.";
```

Reply:
Fixed the typo